### ISSUE TITLE
add native root game folder support for plugins like plugin preloader and script extenders, manual install and management is stupid

### ISSUE THREAD

**Issue by modderchris** [ModOrganizer2/modorganizer#1970]
simple, the "root builder" fails to load with some "not defined" issue, it may be because or portable mode, but that should not affect anything, it should still work
some of us like to try new versions of mods that require new versions of the game WITHOUT having to manually replace our copy of xxSE(fallout or skyrim in my case) so why are we restricted to what goes into the data folder? when a mod that requires game root triggers a "structure invalid" error just have a second symbolic folder that points to game install root, the lack of this option is what makes you just as bad as NMM or vortex, we need root file management, 10 years ago
just add a native folder that doesn't crash MO2 when trying the ".." as a folder name to go up one directory as REQUIRED

**Comment by AnyOldName3** [comment]
There are complicated technical reasons why basically none of the mods that require installing to the root rather than `Data` will work with MO2's VFS (you can basically boil it down to those mods hooking the executable at an earlier point during its launch than USVFS can install its own hooks). That means that without an entirely new VFS, which would be a massive undertaking if it were to be as fast as USVFS is, these mods have to be installed another way. Compared to using the VFS, that's not going to be as good, and there are a few different reasonable approaches, so we've left that up to plugins like root builder. They work fine for loads of people, so if you discuss your root builder issue somewhere the root builder author will see it, they'll almost certainly be able to get it working for you, too.

tl;dr: if it was a native feature rather than a plugin feature, it wouldn't end up working any better than it does as a plugin.
